x-pack/legacy/plugins/reporting/export_types/csv/server/execute_job.js

```diff
@@ -33,6 +34,7 @@ function executeJobFn(server) {
     try {
       decryptedHeaders = await crypto.decrypt(serializedEncryptedHeaders);
     } catch (err) {
+      jobLogger.error(err);
```

Comment:
Not exactly related to this PR, but this really should have been logged previously.